src/index.ts

```diff
@@ -43,6 +44,11 @@ function convert(num: number, type: string) {
     case 'yr':
     case 'y':
       return num * DURATION.YEAR;
+    case 'months':
+    case 'month':
+    case 'mos':
+    case 'mo':
+      return (num * DURATION.YEAR) / 12;
```

Comment:
```suggestion
      return num * DURATION.MONTH;
```

Reply:
That makes sense. I should have seen that.